drivers/nuopc/lnd_comp_driver.F90

```diff
@@ -574,6 +574,7 @@ subroutine drv_run(gcomp, noahmp, rc)
          noahmp%model%julian   , noahmp%domain%garea    , &
          noahmp%model%rainn_mp , noahmp%model%rainc_mp  , &
          noahmp%model%snow_mp  , noahmp%model%graupel_mp, noahmp%model%ice_mp    , &
+         noahmp%model%rhonewsn1, &
```

Comment:
I am not sure how the new variable handled in CCPP/Physics side but I think the cap also need to provide it to be consistent with the CCPP version of the Noah-MP. As I understand from you implementation, the variable is initialized to 0 in component version (NUOPC cap) and does not change but I think in the CCPP/Physics implementation it is chnaging. Since we are comparing component version with the CCPP/Physics version of the Noah-MP this will create difference between them. Is there any specific code in CCPP/Physics side that calculates this new argument. If so, that needs to be ported to NUOCP cap too. Is there any way to calculate it also in NUOPC cap?

Reply:
I think it is calculated under `GFS_MP_generic_post.F90` but not sure how same can be done in component model. @barlage do you have any idea about it? If CCPP/Physics provide it but component version does not then this means that the results will be different in both side.

Reply:
In my understanding, the short term solution is keep this as zero in the component version, this is only needed for one opt_snf option; eventually we will want to bring this through the cap. I don't think that we want to calculate this in the component model or land cap since we will need to pass several fields into the component model to calculate this.

Reply:
@barlage Then, we could merge it as it is. Right?

Reply:
Yes.

Reply:
As long as we don't activate that option in the tests, the results should be identical.